simulators/main/oeeAPI.py

```diff
@@ -167,21 +167,25 @@ def add_timeslot(self, locationId, timeslot):
         log.warning(f'Cannot create Timeslot for location:{locationId}, content: {response.status_code} - {response.text}, url: {url}, data: {json.dumps(timeslot)}')
         return False
 
-    def get_shiftplan(self, locationId, dateFrom, dateTo):
-        url = f'{self.SHIFTPLAN_REST_ENDPOINT}/{locationId}?dateFrom={dateFrom}&dateTo={dateTo}'
-        response = requests.get(url, headers=C8Y_HEADERS)
+    def get_shiftplan_with_specific_date(self, locationId, dateFrom, dateTo):
```

Comment:
- can you use default parameter values, i.e. `def get_shiftplan_with_specific_date(self, locationId, dateFrom=None, dateTo=None)`? this way, you can call it later without supplying date parameters
- maybe we dont need to rename the method if the date parameters are optional?

Reply:
I will set the name back to get_shiftplan, the default value for dateFrom and dateTo are not necessary because we might need to extract shiftplan at specific time range later